schemas/Clinical/index.js

```diff
@@ -292,6 +293,7 @@ const convertClinicalSubmissionEntityToGql = (type, entity) => {
       get(entity, 'dataErrors', []).map(error => convertClinicalSubmissionErrorToGql(error)),
     dataUpdates: () =>
       get(entity, 'dataUpdates', []).map(update => convertClinicalSubmissionUpdateToGql(update)),
+    createdAt: 'DateGoesHERE', // this is a place holder for now
```

Comment:
I think you want `new Date()` cause the `DateTime` type serializes a `Date` to a standard format

Reply:
hmm i tested that and it still worked :thinking:, any ways changed.

Reply:
hmmm gql might be trying to be graceful...